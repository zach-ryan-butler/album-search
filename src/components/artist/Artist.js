import React from 'react';
import PropTypes from 'prop-types';

function Artist({ name, id }) {
  return (
    <section>
      <p>{name}</p>
      <p>{id}</p>
    </section>  
  );
}

Artist.propTypes = {
  name: PropTypes.string.isRequired,
  id: PropTypes.string.isRequired
};

export default Artist;
