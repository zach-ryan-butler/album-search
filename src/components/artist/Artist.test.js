import React from 'react';
import { shallow } from 'enzyme';
import Artist from './Artist';

describe('Artist Component', () => {
  it('renders Artist', () => {
    const wrapper = shallow(<Artist 
      obj={{
        name: 'my name',
        id: 'my id'
      }}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
