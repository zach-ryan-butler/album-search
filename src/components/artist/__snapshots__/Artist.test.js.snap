// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Artist Component renders Artist 1`] = `
<section>
  <Link
    to="/albums/my name/my id"
  >
    my name
  </Link>
</section>
`;
